src/index.d.ts

```diff
@@ -97,48 +97,45 @@ export interface CurrentLocation {
   matchedGeofences: [GeofenceEvent]
 }
 
-export interface PilgrimSdk {
+export default class {
```

Comment:
how does this change the API that people consume?

Reply:
It wouldn't but was a mistake anyways since this class would never be created in js